microsoft-edge/toc.yml

```diff
@@ -24,9 +24,9 @@
 # /top leaf nodes
 # =============================================================================
 # Release notes
-    - name: Release notes
+    - name: Release notes for Microsoft Edge web platform
```

Comment:
```suggestion
    - name: Release notes
```

Reply:
i reverted lengthening toc bucket name, so we can see if we can keep it terse, "Release notes", given that the contained article has been functionally broadened to be sort of global.
@captainbrosset